src/config.rs

```diff
@@ -1907,6 +1910,7 @@ impl Default for TiKvConfig {
             log_file: "".to_owned(),
             slow_log_file: "".to_owned(),
             slow_log_threshold: ReadableDuration::secs(1),
+            panic_log_file: "tikv_stderr.log".to_owned(),
```

Comment:
tidb-ansible creates `tikv_stderr.log` in `./log`, so it will be a different file. That's why I prefer use stderr directly.

Reply:
you can change it in config file or change default value to `./log/tikv_stderr.log`, use tidb-ansible 
or `io redirect` seems to be a better choice, this pr depends on you :).